build-system/tasks/ava.js

```diff
@@ -15,29 +15,30 @@
  */
 'use strict';
 
-const gulp = require('gulp');
-const gulpAva = require('gulp-ava');
+const {exec} = require('../common/exec');
 
 /**
  * Runs ava tests.
- * @return {!Vinyl}
+ * @param {function} cb
  */
-function ava() {
-  return gulp
-    .src([
-      require.resolve('./get-zindex/test.js'),
-      require.resolve('./prepend-global/test.js'),
-    ])
-    .pipe(
-      gulpAva({
-        'concurrency': 5,
-        'failFast': true,
-      })
-    );
+function ava(cb) {
+  const avaCli = 'node_modules/ava/cli.js';
+  const testFiles = [
+    require.resolve('./get-zindex/get-zindex.test.js'),
+    require.resolve('./prepend-global/prepend-global.test.js'),
+  ].join(' ');
+  const {status} = exec(`${avaCli} ${testFiles} --color --fail-fast`);
```

Comment:
why didnt we copy over the `concurrency` option? is it defaulted to something?

Reply:
Yes, it defaults to the number of CPU cores, so setting `--concurrency 5` on an 8 core machine will slow things down. (Not that it matters here, since there are 5 test that run in under 1s.)